includes/admin/functions.php

```diff
@@ -80,18 +80,28 @@ function amp_add_customizer_link() {
 }
 
 /**
- * Registers a top-level menu for AMP configuration options
+ * Registers AMP settings.
  */
 function amp_add_options_menu() {
 	if ( ! is_admin() ) {
 		return;
 	}
 
-	$show_options_menu = apply_filters( 'amp_options_menu_is_enabled', true );
-	if ( true !== $show_options_menu ) {
+	/**
+	 * Filter whether to enable the AMP settings.
+	 *
```

Comment:
Could you find when this filter was introduced and add a `@since` tag for the version?